tt_metal/llrt/tt_cluster.hpp

```diff
@@ -94,6 +94,14 @@ class Cluster {
         return device->get_fast_pcie_static_tlb_write_callable(mmio_device_id);
     }
 
+    tt::Writer get_static_tlb_writer(tt_cxy_pair target) const {
```

Comment:
can you add a comment about when this should be used / scenarios where this could provide perf benefit